pom.xml

```diff
@@ -17,7 +17,7 @@
     <dependency>
       <groupId>org.hibernate</groupId>
       <artifactId>hibernate-core</artifactId>
-      <version>5.4.15.Final</version>
+      <version>5.4.32.Final</version>
```

Comment:
Worth updating the PR title to match the actual change

Reply:
done